tests/foreman/ui/test_oscap_tailoringfiles.py

```diff
@@ -66,20 +82,19 @@ def test_positive_create(self):
         :CaseImportance: Critical
         """
         with Session(self) as session:
-            for tailoringfile_name in valid_data_list():
-                with self.subTest(tailoringfile_name):
+            for tailoring_file_name in valid_data_list():
+                with self.subTest(tailoring_file_name):
                     make_oscap_tailoringfile(
                         session,
-                        name=tailoringfile_name,
+                        name=tailoring_file_name,
                         tailoring_path=self.tailoring_path,
                         tailoring_loc=self.loc_name,
                         tailoring_org=self.org_name,
                     )
                     self.assertIsNotNone(
-                        self.oscaptailoringfile.search(tailoringfile_name))
+                        self.oscaptailoringfile.search(tailoring_file_name))
```

Comment:
Could you add a `msg` parameter with a nice message explaining what has happened? It speeds up the eventual debugging and provides more info than the generic "Unexpectedly None"

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsNotNone

Reply:
@rplevka think if message added we have to assume that the UI has never been changed and this can lead to confusion too, as the element can exist but it was not catched by using the current  locator.
Think in case of this UI assertion fail we have to look at the screenshot anyway despite of the error message (and ignore the message).

Reply:
@ldjebran This is not for me in order to safely say what went wrong. This is mainly for identifying the assert, which failed (and what it was tring to do).

- this allows me distinguish the failures that occur on the specific assert especially in the tests having multiple `AssertNotNone` along.